<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view::MainMenuWindow</class>
 <widget class="QMainWindow" name="view::MainMenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainMenuWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>32</number>
    </property>
    <property name="rightMargin">
     <number>32</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Monopoly</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn2Player">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>2 Player</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn3Player">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>3 Player</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn4Player">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>4 Player</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn5Player">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>5 Player</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn6Player">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>6 Player</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn7Player">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>7 Player</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn8Player">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>8 Player</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnExit">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
